tracee-ebpf/tracee/net_events.go

```diff
@@ -63,17 +62,43 @@ func (t *Tracee) processNetEvents() {
 						t.handleError(err)
 						continue
 					}
+					networkProcess, err := t.getProcessCtx(hostTid)
+					hr, min, sec := timeStampObj.Clock()
+					nsec := uint16(timeStampObj.Nanosecond())
+
+					if err != nil {
+						fmt.Printf("%v:%v:%v:%v  %-16s  %-7d  debug_net/packet               Len: %d, SrcIP: %v, SrcPort: %d, DestIP: %v, DestPort: %d, Protocol: %d\n",
+							hr,
+							min,
+							sec,
+							nsec,
+							comm,
+							hostTid,
+							pktLen,
+							netaddr.IPFrom16(pktMeta.SrcIP),
+							pktMeta.SrcPort,
+							netaddr.IPFrom16(pktMeta.DestIP),
+							pktMeta.DestPort,
+							pktMeta.Protocol)
+					} else {
```

Comment:
are there any cases in which the `t.getProcessCtx(hostTid)` doesn't return the process context?

Reply:
yep `t.getProcessCtx(hostTid)` returns the process context error